sklearn/ensemble/forest.py

```diff
@@ -805,6 +805,10 @@ class RandomForestClassifier(ForestClassifier):
         If None then unlimited number of leaf nodes.
         If not None then ``max_depth`` will be ignored.
 
+    beta : float, optional (default=0.)
+        Threshold for early stopping in tree growth. If the impurity
+        of a node is below the threshold, the node is a leaf.
```

Comment:
Might want to be more explicit here, saying that a node will split if its impurity is above the min_impurity_split, otherwise is a leaf.